sms_logs/src/aws.ts

```diff
@@ -0,0 +1,40 @@
+import l from "aws-lambda";
+import { SecretsManager } from "@aws-sdk/client-secrets-manager";
+
+export function decodeEvent(
+  ev: l.CloudWatchLogsEvent
+): l.CloudWatchLogsDecodedData {
+  const data = ev.awslogs.data;
+  const log = Buffer.from(data, "base64").toString("utf8");
+  const logData: l.CloudWatchLogsDecodedData = JSON.parse(log);
+
+  return logData;
+}
+
+export async function createAlertsConfig() {
+  const client = new SecretsManager({ region: "us-east-1" });
+
+  const endpoints = await Promise.all(
+    ["alerts", "alerts-dev", "alerts-dev-green"].map(async (s) => {
+      const url = `https://${s}.mbtace.com/sms_failure`;
+      const secretName = `${s}-sms-failure-key`;
```

Comment:
Can we make the secret itself available in an environment variable? I know we do this with many of our ECS apps, for example, and that we've been trying to move to that approach over using Secrets Manager as it's much simpler to implement on the app side.

Reply:
Similar concerns here regarding using env vars.